protocol/x/ratelimit/keeper/keeper.go

```diff
@@ -77,6 +77,15 @@ func (k Keeper) ProcessWithdrawal(
 			)
 		}
 
+		metrics.EmitTelemetryWithLabelsForExecMode(
+			ctx,
+			[]sdk.ExecMode{sdk.ExecModeFinalize},
+			metrics.IncrCounterWithLabels,
+			metrics.RateLimitWithdrawalAmount,
+			metrics.GetMetricValueFromBigInt(amount),
```

Comment:
We should not emit `amount` as a label. For some context, search for "infinite cardinality" in Slack

Reply:
amount isnt a label, its a value field